src/datatable.ts

```diff
@@ -772,7 +772,15 @@ export class DataTable {
                 if (ignorePunctuation) {
                     columnQuery = columnQuery.replace(/[.,/#!$%^&*;:{}=-_`~()]/g, "")
                 }
-                return columnQuery
+                const isSplitQueryWord = column.isSplitQueryWord
+                const searchQuerySeparator = column.searchQuerySeparator
+                let columnQueryArr = []
+                if (isSplitQueryWord) {
+                    columnQueryArr = columnQuery.split(searchQuerySeparator)
+                } else {
+                    columnQueryArr.push(columnQuery)
+                } 
+                return columnQueryArr.filter(queryWord => queryWord)
```

Comment:
What are you trying to achieve with this filter? It seems like you are just creating a copy of the area with any potential falsy queryWords removed. Is that the point?